micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/JvmMemory.java

```diff
@@ -41,14 +41,24 @@ private JvmMemory() {
     static boolean isConcurrentPhase(String cause, String name) {
         return "No GC".equals(cause)
                 || "Shenandoah Cycles".equals(name);
+                // Don't think this applies to J9 - maybe "J9MMCONSTANT_IMPLICIT_GC_COMPLETE_CONCURRENT" / concurrent collection must be completed"
     }
 
     static boolean isYoungGenPool(String name) {
-        return name != null && name.endsWith("Eden Space");
+        return name != null && (name.endsWith("Eden Space") 
+            || name.endsWith("nursery-allocate") 
+            || name.endsWith("-survivor") // "nursery-survivor" & "balanced-survivor"
+            || name.endsWith("-eden") // "balanced-eden"
+        );
     }
 
     static boolean isOldGenPool(String name) {
-        return name != null && (name.endsWith("Old Gen") || name.endsWith("Tenured Gen"));
+        return name != null && (name.endsWith("Old Gen")
+            || name.endsWith("Tenured Gen")
+            || name.endsWith("balanced-old")
+            || name.endsWith("balanced-reserved")
```

Comment:
I wasn't sure from reading [this documentation on the balanced collector](https://www.eclipse.org/openj9/docs/gc/#gc-processing_1) how the `balanced-reserved` region was used. I'm trying to figure out how we should calculate the live and max size. For comparison, with OpenJDK collectors we calculate it as the used/max size of the old gen pool after a GC, respectively.

Reply:
I'm still trying to track this one down.  Will update when I get confirmation from the gc devs

Reply:
`balanced-reserved` represents regions in the heap that are always empty.  They'll eventually be used as either eden or  tenure space.

The used space should always be 0 as they are always empty.  The max size should conceptually start at the  entire heap size as nothing will have been allocated yet.

Reply:
If I'm understanding correctly, then it sounds like we shouldn't use it in any of the calculations.

Reply:
I agree.  It might be interesting to track as its own thing but shouldn't be rolled up into any other calculation